recipes/firefox/bld.bat

```diff
@@ -0,0 +1,4 @@
+if not exist %LIBRARY_BIN% mkdir %LIBRARY_BIN% || exit 1
+
+:: https://support.mozilla.org/en-US/kb/deploy-firefox-msi-installers
+start "Install Firefox MSI" /wait msiexec.exe /i firefox.msi INSTALL_DIRECTORY_PATH=%LIBRARY_BIN% TASKBAR_SHORTCUT=false DESKTOP_SHORTCUT=false INSTALL_MAINTENANCE_SERVICE=false /quiet || exit 1
```

Comment:
Ideally, to avoid "polluting" the `=`%LIBRARY_BIN%` directory, we should do something similar to what we do with macOS and Linux. However, this is good enough for now and a future change can happen in the feedstock.